tests/test_search_readonly.py

```
@@ -0,0 +1,812 @@
+        secret_target.mkdir(parents=True)
+        (secret_target / 'classified.pdf').write_text('secret')
+
+        link_path = customer_dir / 'linked_job'
+        try:
+            os.symlink(secret_target, link_path, target_is_directory=True)
+        except (OSError, NotImplementedError):
+            pytest.skip("symlink creation not permitted in this environment")
+        return customer_dir, real_subdir, link_path
+
+    def test_symlinked_subdirectory_excluded_from_readonly_tree(self, tmp_path):
+        customer_dir, real_subdir, link_path = self._make_symlinked_layout(tmp_path)
```

Ruff (0.16.1)

[warning] Unpacked variable `real_subdir` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)